README.md

```diff
@@ -66,6 +66,8 @@ The configuration of the MultiApps CF plugin is done via env variables. The foll
 * `DEBUG=1` - Enables the logging of HTTP requests in `STDOUT` and `STDERRR`.
 * `DEPLOY_SERVICE_URL=<URL>` - By default, the plugin attempts to deduce the multiapps-controller URL based on the available shared domains. In case of issues, or if you want to target a non-default multiapps-controller instance, you can configure the targeted URL via this env variable.
 * `CHUNK_SIZE_IN_MB=<POSITIVE_INTEGER>` - By default, large MTARs are not uploaded as a single unit, but are split up into smaller chunks of 45 MBs that are uploaded separately. The goal is to prevent failed uploads due to [gorouter](https://github.com/cloudfoundry/gorouter)'s request timeout. In case the default chunk size is still too large, you can configure it via this environment variable.
+:rotating_light: Note: If the custom environment variable `CHUNK_SIZE_IN_MB` is specified, the minimum archive chunk size is calculated. If the custom specified chunk size is lower than the minimum archive chunk size, there will be an error displayed. 
```

Comment:
Change to:

Note: The total number of chunks in which an MTAR is split cannot exceed 50, since the multiapps-controller could interpret bigger values as a denial-of-service attack. For this reason, the minimum value for this environment variable is computed based on the formula:
```
MIN = MTAR_SIZE / 50
```
For example, with a 100MB MTAR the minimum value for this environment variable would be `2`, and for a 400MB MTAR it would be `8`. Finally, the minimum value cannot grow over 50, so with a 4GB MTAR, the minimum value would be `50` and **not** `80`.